specs/ExtendedAcceleratorKeyPressed.md

```diff
@@ -0,0 +1,126 @@
+# Background
+WebView2 has `ICoreWebView2AcceleratorKeyPressedEventArgs`  for the `AcceleratorKeyPressed` event. We has been asked extend this API so that developers can indicate that they would like an existing browser accelerator key to be disabled as a browser accelerator.

```

Comment:
```suggestion
WebView2 has `ICoreWebView2AcceleratorKeyPressedEventArgs`  for the `AcceleratorKeyPressed` event. We have been asked to extend this API so that developers can indicate that they would like an existing browser accelerator key to be disabled as a browser accelerator.

```